city_scrapers_core/extensions/status.py

```diff
@@ -146,3 +146,21 @@ def update_status_svg(self, spider: Spider, svg: str):
             ContentType="image/svg+xml",
             Key=f"{spider.name}.svg",
         )
+
+
+class GCSStatusExtension(StatusExtension):
+    """Implements :class:`StatusExtension` for Google Cloud Storage"""
+
+    def update_status_svg(self, spider: Spider, svg: str):
+        """Implements writing templated status SVG to Google Cloud Storage
```

Comment:
Another really small thing because RST is really picky about formatting. I'm not actually sure if the suggestion will work the way I think here and add a newline, but that's the only change it needs

```suggestion
        """Implements writing templated status SVG to Google Cloud Storage

```